frontend/src/views/SQR/Form.vue

```diff
@@ -321,6 +318,7 @@ export default class Form extends Vue {
         this.initialData[".value"] !== null ||
         branch === SubmissionsBranch.DRAFTS
       ) {
+        this.branch = branch;
```

Comment:
Seems like this is the only place where `branch` is set. What about the following scenario:

- The form is loaded from the `drafts` branch.
- The form is submitted by the user.
- From now on the `branch` should be `completed`, not `drafts`.

Is it covered?

Reply:
After the form is submitted a different screen is shown and to get back to the form a page refresh is needed so there's no need to assign `this.branch` to `completed`.

Reply:
I see.